single-node/user-data

```diff
@@ -104,10 +116,64 @@ ExecStart=/usr/lib/coreos/kubelet-wrapper \
   --cluster_domain=cluster.local
 Restart=always
 RestartSec=10
+KillMode=process
+
+[Install]
+WantedBy=multi-user.target
+EOF
+    }
+
+    local TEMPLATE=/etc/systemd/system/install-kproxy.service
+    [ -f $TEMPLATE ] || {
+        echo "TEMPLATE: $TEMPLATE"
+        mkdir -p $(dirname $TEMPLATE)
+        cat << EOF > $TEMPLATE
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStartPre=/usr/bin/mkdir -p /opt/
+ExecStartPre=/usr/bin/curl --fail --silent --show-error --location --create-dirs --output /opt/kube-proxy https://storage.googleapis.com/kubernetes-release/release/v1.3.0-alpha.5/bin/linux/amd64/kube-proxy
```

Comment:
Probably, but i'm hoping the remove this work-around still. Noted that if we end up leaving this in, I'll make sure we configure the version.